### PR TITLE
Update comments and fix some functionality

### PR DESCRIPTION
**Pull request by joecohens**
/cc @arturock

### CODE REVIEW

File: src/Gateways/Conekta.php [hunk 1]
```diff
@@ -379,20 +379,20 @@ protected function parseResponse($body)
      */
     protected function responseError($rawResponse)
```
Comment (arturock) [inline src/Gateways/Conekta.php]: maybe $responseBody?

File: src/Gateways/ConektaOxxo.php [hunk 2]
```diff
@@ -18,8 +18,8 @@ class ConektaOxxo extends Conekta
     /**
      * Charge the credit card.
```
Comment (arturock) [inline src/Gateways/ConektaOxxo.php]: Other docs are in third p

Comment (joecohens) [inline src/Gateways/ConektaOxxo.php]: Yep, not this one haha

File: src/Contracts/Store.php [hunk 3]
```diff
@@ -10,7 +10,7 @@
      * @param $creditcard
```
Comment (arturock) [inline src/Contracts/Store.php]: mixed?

File: src/Gateways/Conekta.php [hunk 4]
```diff
@@ -91,10 +93,10 @@ public function charge($amount, $payment, $options = [])
     /**
      * Stores a credit card.
      *
-     * @param $creditcard
+     * @param string   $creditcard
```
Comment (arturock) [inline src/Gateways/Conekta.php]: Maybe $cardToken would be better suited

Comment (joecohens) [inline src/Gateways/Conekta.php]: Noup, breaks the interface.

Comment (arturock) [inline src/Gateways/Conekta.php]: Got it

File: src/Gateways/Conekta.php (hunk 1 above)
Comment (joecohens) [inline src/Gateways/Conekta.php]: :+1:

**Pull request merged**